### ISSUE TITLE
gmp_random_seed return value inconsistency

### ISSUE THREAD

**Issue by AndrolGenhald** [php/doc-en#1387]
https://www.php.net/manual/en/function.gmp-random-seed.php

The description has `: void` but the Return Values section says "Returns null on success or false on failure."

It looks like a changelog needs added to note the change from false return to error: https://3v4l.org/pokBp

**Comment by cmb69** [comment]
Indeed, another case of `ValueError` as of PHP 8.0.0 (the `TypeError` in 8.0.0 - 8.0.1 was a bug which doesn't need to be documented).
